Rewrite `osu!academy` (#5931)
ppy/osu-wiki: wiki/osu!academy/en.md

# osu!academy

![](img/logo.png "osu!academy logo")

> The **osu!academy** is a video series hosted by the [osu! team](/wiki/People/The_Team) that aims to give a comprehensive look at all aspects of osu!, from playing to mapping and everything in between.

Reviewer: ```suggestion
**osu!academy** is a video series hosted by the [osu! team](/wiki/People/The_Team) that aims to give a comprehensive look at all aspects of osu!, from playing to mapping and everything in between.
```
nit; "the" is probably not needed here

Author: should be okay as it is i think

> Throughout the years, the osu!academy has been hosted by multiple members of the community, with the most prominent one being [ztrot](https://osu.ppy.sh/users/6347), who has maintained the series for many years and is the face of the project.

Reviewer: ```suggestion
Throughout the years, osu!academy has been hosted by multiple members of the community, the most prominent of which being [ztrot](https://osu.ppy.sh/users/6347), who has maintained the series for many years and is the face of the project.
```
also a nitpick

## Main series

The main series of osu!academy serves as a general introduction to the broad world of osu!. It aims to familiarise users with all major features of osu! and its services, which might confuse newcomers due to a steep learning curve associated with the game.

*Note: Episodes 1-6 have been rerecorded, as they presented obsolete information, and thus were deemed as unhelpful.*

| No. | Title | Description |
| :-: | :-- | :-- |
| 0 | [Lazer installer setup guide](https://www.youtube.com/watch?v=kO5ISAzRbPE "YouTube") | [ztrot](https://osu.ppy.sh/users/6347) covers the process of installing the osu!lazer client. |
| 1 | [osu! Installation/Account Setup/Beatmap Downloads](https://www.youtube.com/watch?v=0V5GwzmMhpU "YouTube") | [pishifat](https://osu.ppy.sh/users/3178418) explains how to install osu!, create an account, and download new beatmaps. |

> | 2 | [Standard Gameplay/Settings Adjustments](https://www.youtube.com/watch?v=mswLEXK0eDk "YouTube") | [ztrot](https://osu.ppy.sh/users/6347) goes over the osu! mode, and explains how to adjust gameplay settings. |

Author: I think most of them are oxford commas so i will leave them as they are

also I would leave `mode` instead of `game mode` because it sounds more natural for me

| 3 | [Honeycomb patterns](https://www.youtube.com/watch?v=_hla2JtEPkU "YouTube") | [ztrot](https://osu.ppy.sh/users/6347) covers the positioning and use of the Honeycomb patterns. |
| 4 | [Star patterns](https://www.youtube.com/watch?v=LHch8V-v7i4 "YouTube") | [ztrot](https://osu.ppy.sh/users/6347) explains how to create Star patterns, and covers the shortcuts associated with them. |
| 5 | [Compressed Streams](https://www.youtube.com/watch?v=2XtP58p9DtY "YouTube") | [ztrot](https://osu.ppy.sh/users/6347) demonstrates the process of creating Compressed Streams using distance spacing and snaps. |
| 6 | [Sliderstream Jumps](https://www.youtube.com/watch?v=tPRFa2sRXSs "YouTube") | [ztrot](https://osu.ppy.sh/users/6347) covers the usage of Sliderstream Jumps, a new trend that has appeared in mapping. |
| 7 | [Variable Streams](https://www.youtube.com/watch?v=oaIWbuEihZk "YouTube") | [ztrot](https://osu.ppy.sh/users/6347) goes through the arrangement and usage of the Variable Streams. |

## Time Lapse Mapping

> **[Time Lapse Mapping](https://www.youtube.com/playlist?list=PLmWVQsxi34bO7PAmjWZjVDi16pAy-GRBP "YouTube")** is a series of videos hosted by [ztrot](https://osu.ppy.sh/users/6347) and several mappers from different game modes, where they explain the process of creating beatmaps with a sped-up mapping session in the background.

Reviewer 2: ```suggestion
**[Time Lapse Mapping](https://www.youtube.com/playlist?list=PLmWVQsxi34bO7PAmjWZjVDi16pAy-GRBP "YouTube")** is a series of videos hosted by [ztrot](https://osu.ppy.sh/users/6347) and several mappers of different game modes, where they explain the process of creating beatmaps with a sped-up mapping session in the background.
```
